### ISSUE TITLE
How do you get those four npy files?

### ISSUE THREAD

**Issue by wusaifei** [zhangboshen/Alibaba-Cloud-German-AI-Challenge-2018-Rank-17th-Solution#1]
Hello, I'd like to ask how do you get those four npy files? Have you generated the 4 npy files or ideas, please answer. Thank you very much for your answer.

**Comment by zhangboshen** [comment]
Yes, I assume that you asked this same question on the TianChi forum (if not, never mind.).
This four npy files, we obtained by computing mean and std using the whole training set. Corresponding to mean and std for sensor1 inputs and sensor2 inputs.

**Comment by zhangboshen** [comment]
> Hello, I'd like to ask how do you get those four npy files? Have you generated the 4 npy files or ideas, please answer. Thank you very much for your answer.

Yes, I assume that you asked this same question on the TianChi forum (if not, never mind.).
This four npy files, we obtained by computing mean and std using the whole training set. Corresponding to mean and std for sensor1 inputs and sensor2 inputs.

**Comment by wusaifei** [comment]
Thank you very much for your reply to my message. The one above Tianchi is also my problem. Can you privately distribute the program that generates npy if you can? I am a beginner of machine learning. I can't write your calculation process. I want to refer to the code you wrote. Thank you very much.

**Comment by wusaifei** [comment]
@zhangboshen Thank you very much for your reply to my message. The one above Tianchi is also my problem. Can you privately distribute the program that generates npy if you can? I am a beginner of machine learning. I can't write your calculation process. I want to refer to the code you wrote. Thank you very much.

**Comment by wusaifei** [comment]
@zhangboshen  My mailbox is  wusaifei@foxmail.com
